### PR TITLE
Feat: Added Viewing Team Member Popup 

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description

Fixes:- #1146 

In this PR i have added a modal which appear when user clicks on particular member in members tab inside team page , also updated the UI part for the modal .

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/packages/@library/ui/modal/Modal.svelte [hunk 1]
```diff
@@ -68,13 +86,23 @@
...
-    background-color: var(--bg-tertiary-750);
+    right: 0;
+    bottom: 0;
+    top: 0;
+    left: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    /* background-color: var(--bg-tertiary-400); */
+    border-radius: 6px;
```
Comment (itsmdasifraza) [inline src/packages/@library/ui/modal/Modal.svelte:101]: comment not required here

File: src/packages/@library/icons/Arrow.svelte [hunk 2]
```diff
@@ -1,12 +1,18 @@
+<script>
+  export let width = 14;
+  export let height = 16;
+  export let color = "white";
+</script>
```
Comment (itsmdasifraza) [inline src/packages/@library/icons/Arrow.svelte:5]: use 24 24 grey

File: src/packages/@library/forms/select/Select.svelte [hunk 3]
```diff
@@ -445,20 +445,31 @@
...
       </p>
       <span class="d-flex ps-2" class:select-logo-active={isOpen}>
         {#if isDropIconFilled}
-          <ArrowIcon />
+          <ArrowIcon
+          width={12}
+          height={14}
+            color={disabled || selectedRequest?.hide
+              ? "var(--text-secondary-370)"
+              : "var(--sparrow-text-color)"}
```
Comment (itsmdasifraza) [inline src/packages/@library/forms/select/Select.svelte:464]: icon color should start with --icon

File: src/packages/@library/forms/select/Select.svelte [hunk 4]
```diff
@@ -445,20 +445,31 @@
...
+          />
         {:else}
           <DownArrowIcon
             width={12}
             height={14}
-            color={"var(--sparrow-text-color)"}
+            color={disabled || selectedRequest?.hide
+              ? "var(--text-secondary-370)"
+              : "var(--sparrow-text-color)"}
```
Comment (itsmdasifraza) [inline src/packages/@library/forms/select/Select.svelte:472]: icon color should start with --icon

**Review by itsmdasifraza — CHANGES_REQUESTED** [review]

File: src/packages/@library/ui/modal/Modal.svelte [hunk 5]
```diff
@@ -24,40 +24,58 @@
...
   </div>
 {/if}
 
 <style lang="scss">
+  .testing-desgin {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
```
Comment (itsmdasifraza) [inline src/packages/@library/ui/modal/Modal.svelte:77]: class name should be more better

**Review by itsmdasifraza — CHANGES_REQUESTED** [review]

**Comment by itsmdasifraza** [comment]
use issue hash in comment, please don't leave PR description blank

File: src/packages/@library/ui/modal/Modal.svelte (hunk 1 above)
Comment (Abhishek9503) [reply src/packages/@library/ui/modal/Modal.svelte:101]: Resolved

File: src/packages/@library/icons/Arrow.svelte (hunk 2 above)
Comment (Abhishek9503) [reply src/packages/@library/icons/Arrow.svelte:5]: Resovled

File: src/packages/@library/ui/modal/Modal.svelte (hunk 5 above)
Comment (Abhishek9503) [reply src/packages/@library/ui/modal/Modal.svelte:77]: Resolved

File: src/packages/@library/forms/select/Select.svelte (hunk 4 above)
Comment (Abhishek9503) [reply src/packages/@library/forms/select/Select.svelte:472]: Resolved

File: src/packages/@library/forms/select/Select.svelte (hunk 3 above)
Comment (Abhishek9503) [reply src/packages/@library/forms/select/Select.svelte:464]: Resolved

**Comment by Abhishek9503** [comment]
> use issue hash in comment, please don't leave PR description blank

Resovled

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**